artemis-unit-test-support/src/main/java/org/apache/activemq/artemis/utils/ThreadLeakCheckRule.java

```diff
@@ -295,6 +295,8 @@ private boolean isExpectedThread(Thread thread) {
          return true;
       } else if (threadName.contains("GC Daemon")) {
          return true;
+      } else if (threadName.contains("org.eclipse.jetty.util.RolloverFileOutputStream")) {
```

Comment:
Isn't this hiding a thread leakage? can't we just close the thread instead?

Reply:
Or move this test to the isolated tests if you can't figure out how to close the thread.

Reply:
I don't mean to move the whole test into integration-tests-isolated. but rather write just this test into the isolated package.. 


This is definitely causing a thread leakage.. and this thread could be bothering other tests.

I strongly recommend against it.

Reply:
The thread is specific to the HTTP request logging from Jetty. Looking at the Jetty source code there appears to be no way to close it. I'll move the test into the isolated module.